client/src/ressources/feather.svg

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 512 512" style="enable-background:new 0 0 512 512;" xml:space="preserve">
+<g>
+	<g>
+		<polygon points="438.527,207.166 326.104,207.166 160.128,373.142 272.552,373.142 287.067,358.626 283.067,311.963 
+			329.731,315.963 		"/>
+	</g>
+</g>
+<g>
+	<g>
+		<polygon points="130.048,403.222 100.167,433.103 211.999,433.695 242.471,403.222 		"/>
+	</g>
+</g>
+<g>
+	<g>
+		<polygon points="78.305,300.001 78.897,411.833 223.3,267.43 223.3,155.006 		"/>
+	</g>
+</g>
+<g>
+	<g>
+		<path d="M484.311,27.689c-36.919-36.919-96.775-36.918-133.693,0L306.51,71.797l4,46.663l-46.663-4l-10.466,10.466V237.35
+			l125.211-125.211l21.27,21.27l-43.677,43.678h112.424l15.703-15.703C521.23,124.464,521.23,64.607,484.311,27.689z"/>
+	</g>
+</g>
+<g>
+	<g>
+		<polygon points="79.009,432.991 78.897,411.833 0,490.73 21.27,512 100.167,433.103 		"/>
+	</g>
+</g>
+<g>

```

Comment:
You can remove these empty `g`

Reply:
I decided to use another svg, so I will remove the file.